src/EditorFeatures/Core.Wpf/InlineRename/UI/Dashboard/RenameDashboard.xaml.cs

```diff
@@ -243,7 +244,7 @@ protected override void OnAccessKey(AccessKeyEventArgs e)
                 }
                 else if (string.Equals(e.Key, RenameShortcutKey.Apply, StringComparison.OrdinalIgnoreCase))
                 {
-                    this.Commit();
+                    _ = this.CommitAsync();
```

Comment:
same issue.  also not reporting an error here.

Reply:
`CommitAsync`  itself has a try-catch block which report the error. So not needed here

Reply:
please doc CommitAsync as non-throwing.

Reply:
Added.